Sources/Generics/GenericTableViewController.swift

```diff
@@ -26,7 +26,7 @@ UISearchResultsUpdating, UISearchControllerDelegate {
     /// Enable the search controller
     private var shouldDisplaySearchBar = false
     /// The amount of items to enable the search bar to the Generic TableView
-    let amountOfItemsToEnableSearchBar = 20
+    let amountOfItemsToEnableSearchBar = 4
```

Comment:
Revert this please.  As a note, you never want to be modifying the source code to make your test case pass (with the exception being accessibility identifiers)